workflow/sideload/launch.sh

```diff
@@ -84,7 +84,7 @@ case ${task_id} in
       module load libjpeg/9c
       module load libfabric/1.20.1
     else
-      module load "${MACHINE}"
+      module load "${MACHINE}_intel"
```

Comment:
@EricJames-NOAA could you expand why we need this change? Thanks!

Reply:
@guoqing-noaa yes, the modulefile name was changed in UPP repo.  So we need to load the new modulefile in order to set the environment for UPP.

Reply:
@SamuelDegelia-NOAA good catch, thanks.  I just made that change.